Framework/Algorithms/CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ set ( SRC_FILES
 	src/CorelliCrossCorrelate.cpp
 	src/CorrectFlightPaths.cpp
 	src/CorrectKiKf.cpp
+        src/CorrectTOFAxis.cpp
```

Comment:
These should be tabs instead of spaces to be consistent with the rest of the file.